schema-client-example/src/main/resources/application.yaml

```diff
@@ -0,0 +1,7 @@
+server:
+  port: 8082
+
+miranum:
```

Comment:
Not sure how this made it through the PR. Why are we using a yaml file here but else where a application.properties? Nothing you @Hafflgav made, I know. Just curious

Reply:
That's actually a god topic! We should move everything to `application.yaml`'s since this is more readable if the project gets bigger and more configuration parameters exist. 

We recently did a similar refactoring for miranum-ide